src/main/java/com/google/sps/servlets/ImageHandlerServlet.java

```diff
@@ -79,6 +80,7 @@ private Entity newPlayerEntity(String imageBlobKeyString, String displayName) {
     player.setProperty(IMAGE_ID_PARAMETER, imageBlobKeyString);
     player.setProperty(DISPLAY_NAME_PARAMETER, displayName);
     player.setProperty(ID_PARAMETER, user.getUserId());
+    player.setProperty(EXPERIENCE_PARAMETER, 0);
```

Comment:
nit, but to make it more clear you could have a private static variable like
private static final int NO_EXPERIENCE = 0;